desktop/core/src/desktop/conf.py

```diff
@@ -977,6 +977,13 @@ def is_gunicorn_report_enabled():
       type=str,
       default="OPTIONAL",
     ),
+    HOSTNAME_OVERRIDE=Config(
+      key='hostname_override',
```

Comment:
Nit: Let's add this config in our config files as well? Just like below:
https://github.com/cloudera/hue/blob/0ae016de105f0e6682b6da741e4af9c417adc6a3/desktop/conf.dist/hue.ini#L756
https://github.com/cloudera/hue/blob/0ae016de105f0e6682b6da741e4af9c417adc6a3/desktop/conf/pseudo-distributed.ini.tmpl#L757